### ISSUE TITLE
Cannot rotate text

### ISSUE THREAD

**Issue by boygirl** [FormidableLabs/victory-native#103]
Rotated text elements are also translated. Here's an example with a 90 degree rotation:
<img width="320" alt="screen shot 2017-06-13 at 3 34 24 pm" src="https://user-images.githubusercontent.com/3719995/27107649-1251667c-504e-11e7-9642-d2d2d4846198.png">

This issue prevents the implementation of polar charts, and breaks charts for any users who are rotating their own labels

See https://github.com/react-native-community/react-native-svg/issues/242

**Comment by ElieMyIdea** [comment]
+1

**Comment by boygirl** [comment]
Issue persists in `react-native-svg@6.0.0` :(

**Comment by boygirl** [comment]
Temporary fix in `victory-native@~0.16.0`

This fix uses a manual matrix transformation to rotate labels, but it is not compatible with other (user supplied) transformations. 

A better fix will be implemented pending the resolution of [this issue](https://github.com/react-native-community/react-native-svg/issues/242#issuecomment-344111728)

**Comment by hrmoller** [comment]
@boygirl would you be able to exemplify *manual matrix transformation*?
Specifically I want to rotate my x-axis labels 35 degrees and doing it this way has no effect:
```
<VictoryAxis
  tickFormat={t => this.props.xAxisTickFormatter(t)}
  tickLabelComponent={<VictoryLabel angle={35} />}
/>
```

**Comment by boygirl** [comment]
This is fixed in `0.16` and above
